apps/files/src/components/FileDetails.vue

```diff
@@ -62,9 +62,22 @@ export default {
     fileSideBarsEnabled () {
       return this.fileSideBars.filter(b => b.enabled === undefined || b.enabled(this.capabilities, this.highlightedFile))
     },
+    defaultTab () {
+      return this.fileSideBarsEnabled[0].app
```

Comment:
will explode as soon as there are 0 tabs

Reply:
Added condition to check if there is some tab enabled.

Maybe would be worth to consider if we want to show the sidebar at all in case there would be no tabs. User would end up then with only the name, size and modification date there which doesn't really seem so necessary to be displayed again on its own to me.

Reply:
worth a thought indeed .....